srcpkgs/removed-packages/template

```diff
@@ -417,6 +417,8 @@ replaces="
  zuluplay-devel<=1.1_1
  zuluplay<=1.1_1
  bgc<=1.0.1_1
+ guile1.8-devel<=1.8.8_3
+ guile1.8<=1.8.8_3
```

Comment:
could you move this up to keep this list at least a bit alphabetically sorted?

Reply:
Sure and done. :)